Topics/Topic_2_b/src/main/java/com/bootcamp/topic2_b/blog/Entry.java

```diff
@@ -0,0 +1,90 @@
+package com.bootcamp.topic2_b.blog;
+
+import java.util.Date;
+
+public class Entry {
+	private int id;
+	private Date date;
+	private String title;
+	private String body;
+	private String author;
+	private static int lastId;
+	
+	public Entry (Date date, String title, String author, String body){
+		this.id = lastId++;
+		this.date=date;
+		this.title=title;
+		this.author = author;
+		this.body = body;
+	}
+
+	public Date getDate() {
+		return date;
+	}
+
+	public String getTitle() {
+		return title;
+	}
+
+	public String getBody() {
+		return body;
+	}
+
+	public String getAuthor() {
+		return author;
+	}
+
+	public void setDate(Date date) {
+		this.date = date;
+	}
+
+	public void setTitle(String title) {
+		this.title = title;
+	}
+
+	public void setBody(String body) {
+		this.body = body;
+	}
+
+	public void setAuthor(String author) {
+		this.author = author;
+	}
+
+	public int getId() {
+		return id;
+	}
+	
+	@Override
+	public boolean equals(Object obj) {
+		if (this == obj)
+			return true;
+		if (obj == null)
+			return false;
+		if (getClass() != obj.getClass())
+			return false;
+		Entry other = (Entry) obj;
+		if (author == null) {
+			if (other.author != null)
+				return false;
+		} else if (!author.equals(other.author))
+			return false;
+		if (body == null) {
+			if (other.body != null)
+				return false;
+		} else if (!body.equals(other.body))
+			return false;
+		if (date == null) {
+			if (other.date != null)
+				return false;
+		} else if (!date.equals(other.date))
+			return false;
+		if (id != other.id)
+			return false;
+		if (title == null) {
+			if (other.title != null)
+				return false;
+		} else if (!title.equals(other.title))
+			return false;
+		return true;
+	}
+}
```

Comment:
This class is OK

BlogList isn't.... we have a blog, with a list of entries. (and crud operations with that entity)
not need to implement our linkedlist yet, several bad codes there.... 
remember :: one Blog, that have a list of entries. Then, some methods to do the operations (and testscases scenario for that methods)

chat me in skype for any concern.